glambie/processing/processing_helpers.py

```diff
@@ -143,12 +142,14 @@ def convert_datasets_to_longterm_trends(data_catalogue: DataCatalogue, year_type
     ----------
     data_catalogue : DataCatalogue
         data catalogue to be converted
-
     year_type : YearType
         type of annual year when longterm timeseries should start and end, e.g hydrological or calendar
-
     season_calibration_dataset: Timeseries
         Timeseries dataset for seasonal calibration if trends are at lower resolution than 1 year.
+    min_max_time_window : Tuple[float, float], optional
+        Tuple[min_start_date, max_end_date]
+        If specified, the longterm trend will not start earlier than this date min_start_date and
```

Comment:
Again what format is expected for the date? Float is ambiguous for a date format.

Reply:
i specified